ext/ImplicitDifferentiationForwardDiffExt.jl

```diff
@@ -0,0 +1,52 @@
+module ImplicitDifferentiationForwardDiffExt
+
+@static if isdefined(Base, :get_extension)
+    using ForwardDiff: Dual, Partials, jacobian, partials, value
+else
+    using ..ForwardDiff: Dual, Partials, jacobian, partials, value
+end
+
+using AbstractDifferentiation: ForwardDiffBackend, lazy_jacobian
+using ImplicitDifferentiation: ImplicitFunction, SolverFailureException, LazyJacobianMul!
+using LinearOperators: LinearOperator
+
+"""
```

Comment:
Ah I see why we don't need ForwardDiffChainRules.

Reply:
Yup, I figured it's cleaner this way